client/src/setup/setupLanguageServerHooks.ts

```diff
@@ -48,8 +48,10 @@ const startLanguageServer = (extensionState: ExtensionState): void => {
         workspace.createFileSystemWatcher("**/hardhat.config.{ts,js}"),
         workspace.createFileSystemWatcher("**/{truffle-config,truffle}.js"),
         workspace.createFileSystemWatcher("**/foundry.toml"),
+        workspace.createFileSystemWatcher("**/ape-config.yaml"),
         workspace.createFileSystemWatcher("**/remappings.txt"),
         workspace.createFileSystemWatcher("**/*.sol"),
+        workspace.createFileSystemWatcher("**/.cache/*.sol"),
```

Comment:
It's multiple directories deep. But I just noticed this line can be removed, since the previous one will trigger for sol files inside `.cache` as well.